lib/statsd/instrument.rb

```diff
@@ -356,4 +356,5 @@ def collect_metric(options)
 require 'statsd/instrument/backend'
 require 'statsd/instrument/assertions'
 require 'statsd/instrument/environment'
+require 'statsd/instrument/matchers'
```

Comment:
This means RSpec becomes a required dependency for this gem. That would be a :-1: for me.

This can be fixed easily do:

``` ruby
require 'statsd/instrument/matchers' if defined?(::RSpec)
```